dvc/command/lock.py

```diff
@@ -37,24 +37,28 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    LOCK_HELP = "Lock DVC-file."
+    LOCK_HELP = "Lock DVC-files."
     lock_parser = subparsers.add_parser(
         "lock",
         parents=[parent_parser],
         description=append_doc_link(LOCK_HELP, "lock"),
         help=LOCK_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    lock_parser.add_argument("targets", nargs="+", help="DVC-files.")
+    lock_parser.add_argument(
+        "targets", nargs="*", help="DVC-files to lock. Optional."
```

Comment:
Wait, but why did you change it to "*" here? "+" should work correctly, and https://github.com/iterative/dvc/issues/2199 is caused by your * here.

Reply:
Didn't mean to change any `nargs`. Accidental copy-paste change, sorry! Reverted all in 5ba3b547